internal/db/description/collection.go

```
@@ -136,6 +184,15 @@ func GetCollectionsByCollectionID(
 	ctx context.Context,
 	collectionID string,
 ) ([]client.CollectionVersion, error) {
+	cache := getCollectionCache(ctx)
+	if cache.IsFullyPopulated {
+		return cache.CollectionsByID[collectionID], nil // todo - handle panic?
```

question: if the panic mentioned here is from a nil map, should the map be pre-initialized?

---

sorry - this todo should not have made it in to the PR.  Is a WIP comment.

The map should never be nil, however it will panic if an unknown collection id is provided (via user call, not possible internally).

I'll sort it out shortly, thanks for spotting.

- [x] Sort out todo (might be a second one somewhere too)